deselected_tests.yaml

```diff
@@ -296,17 +300,17 @@ deselected_tests:
   # RandomForestRegressor sum(y_pred)!=sum(y_true)
   - ensemble/tests/test_forest.py::test_balance_property_random_forest[squared_error] >=1.0
 
-  # This tests fail on mac mini 8.1 with stock scikit-learn
+  # This tests fail on mac mini 8.1 with stock sklearn
   - semi_supervised/tests/test_label_propagation.py
 
   # This test fails because with patch config_context with new options, but the
-  # test checks that only exact number of options is used
+  # test checks only the exact number of options that are used
   - tests/test_config.py::test_config_context
 
   # HalvingGridSearchCV with Ridge and PCA didn't have feature_name_in. Need to fix.
   - tests/test_common.py::test_pandas_column_name_consistency >=1.0.1
 
-  # Some sklearnex docstrings differ from scikit-learn.
+  # Some sklearnex docstrings differ from sklearn.
```

Comment:
I am not sure about this changes. It is better leave it as is.

Reply:
I believe it would be good to go with one or the other here. Let me know which you prefer.

Reply:
From my point of view it is better use `scikit-learn`, `scikit-learn-intelex`, as these are package names